response.go

```diff
@@ -178,12 +180,28 @@ func (resp *Response) decodeBody() (err error) {
 				if err = d.Decode(&resp.MetaData); err != nil {
 					return err
 				}
+			case KeyStmtID:
+				if stmtID, err = d.DecodeUint64(); err != nil {
+					return err
+				}
+			case KeyBindCount:
+				if bindCount, err = d.DecodeUint64(); err != nil {
+					return err
+				}
 			default:
 				if err = d.Skip(); err != nil {
 					return err
 				}
 			}
 		}
+		if stmtID != 0 {
+			stmt = &Prepared{
```

Comment:
```suggestion
			stmt := &Prepared{
```
and remove a definition above.

Reply:
Fixed.